src/basic/group_node.cpp

```diff
@@ -166,6 +182,8 @@ class GroupNode {
   std::shared_ptr<trajectory::Trajectory> trajectory_;
   double trajectory_start_time_{ std::numeric_limits<double>::quiet_NaN() };
   double last_time_ = 0;
+  double last_setpoint_time_ = 0;
+  double message_timeout_;
```

Comment:
Minor, since this is set by all constructors right now, but it is best practice to go ahead and set a default value here, even if you have this defined in all current constructors.  Otherwise, this can get initialized with garbage if you add a new constructor and forget to set it.

`double message_timeout_ = 0;` should do the trick.

Reply:
Fixed, thanks!